src/coreclr/jit/clrjit.natvis

```diff
@@ -0,0 +1,134 @@
+    <DisplayString>[{lvType,en}-{lvReason,s}]</DisplayString>
+  </Type>
+
+  <Type Name="GenTreeLclVar" Inheritable="false">
+    <DisplayString>[{gtOper,en}, {gtType,en} V{((GenTreeLclVar*)this)-&gt;_gtLclNum,u}]</DisplayString>
+  </Type>
+
+  <!-- Register allocation -->
+  <Type Name="LinearScan">
+    <DisplayString>LinearScan</DisplayString>
+    <Expand>
+        <Item Name="AvailableRegs mask">this-&gt;m_AvailableRegs</Item>
+        <CustomListItems>
+          <Variable Name="reg" InitialValue="this->m_AvailableRegs" />
+          <Variable Name="regIndex" InitialValue="0" />
+          <Loop Condition="reg != 0">
+            <Item Condition="(reg &amp; 1) != 0">((regNumber)regIndex),en</Item>
+            <Exec>regIndex++</Exec>
+            <Exec>reg = reg >> 1</Exec>
+          </Loop>
+        </CustomListItems>
+        <Item Name="RegistersWithConstants mask">this-&gt;m_RegistersWithConstants</Item>
+        <CustomListItems>
+          <Variable Name="reg" InitialValue="this->m_RegistersWithConstants" />
+          <Variable Name="regIndex" InitialValue="0" />
+          <Loop Condition="reg != 0">
+            <Item Condition="(reg &amp; 1) != 0">((regNumber)regIndex),en</Item>
+            <Exec>regIndex++</Exec>
+            <Exec>reg = reg >> 1</Exec>
+          </Loop>
+        </CustomListItems>
+    </Expand>
+  </Type>
+
+  <Type Name="RefPosition">
+    <DisplayString>[#{rpNum,d} - {refType,en}]</DisplayString>
+    <Expand>
+        <Item Name="Referent" Condition="this->isPhysRegRef">(RegRecord*)this-&gt;referent</Item>
+        <Item Name="Referent" Condition="!this->isPhysRegRef">(Interval*)this-&gt;referent</Item>
+        <CustomListItems>
+          <Variable Name="reg" InitialValue="this->registerAssignment" />
+          <Variable Name="regIndex" InitialValue="0" />
+          <Loop Condition="reg != 0">
+            <Item Condition="(reg &amp; 1) != 0">((regNumber)regIndex),en</Item>
+            <Exec>regIndex++</Exec>
+            <Exec>reg = reg >> 1</Exec>
+          </Loop>
+        </CustomListItems>
+    </Expand>
+  </Type>
+
+  <Type Name="Interval">
+    <DisplayString Condition="this->isLocalVar">[V{this->varNum,d}, #{this->intervalIndex, d}, reg={(regNumber)physReg, en}]</DisplayString>
+    <DisplayString Condition="this->isConstant">[C{this->intervalIndex, d}, reg={(regNumber)physReg, en}]</DisplayString>
+    <DisplayString>[I{this->intervalIndex, d}, reg={(regNumber)physReg, en}]</DisplayString>
+  </Type>
+
+  <Type Name="RegRecord">
+    <DisplayString>[reg={((regNumber)regNum),en}, type={registerType, en}]</DisplayString>
+    <Expand>
+      <Item Name="[Assigned]">assignedInterval</Item>
+      <Item Name="[Previous]">previousInterval</Item>
+    </Expand>
+  </Type>
+
+  <!-- Emitter -->
+  <Type Name="insGroup">
+    <DisplayString Condition="igFlags &amp; 0x200">IG{igNum,d} [extend]</DisplayString>
+    <DisplayString>IG{igNum,d]</DisplayString>
```

Comment:
`]` => `}`

Reply:
@kunalspathak Looks like you missed this one

Reply:
ah, good catch.